classes/class-p4-campaigns.php

```diff
@@ -250,13 +250,13 @@ public function save_taxonomy_meta( $term_id ) {
 				$tag_data = get_term( $term_id );
 
 				if ( $tag_data instanceof \WP_Term ) {
-					$post_content = '[shortcake_newcovers cover_type="1" tags="' . $term_id . '" covers_view="0" title="' . __( 'Things you can do', 'planet4-master-theme' ) . '"  description="' . __( 'We want you to take action because together we\'re strong.', 'planet4-master-theme' ) . '" /]
+					$post_content = '<!-- wp:planet4-blocks/covers {"title":"' . __( 'Things you can do', 'planet4-master-theme' ) . '","description":"' . __( 'We want you to take action because together we\'re strong.', 'planet4-master-theme' ) . '","tags":[' . $term_id . '],"posts":[],"post_types":[],"cover_type":"1","covers_view":"0"} /-->
```

Comment:
same here, please populated an array with the attributes and then generate the json body for the block